README.md

```diff
@@ -50,7 +50,7 @@ gaxios.request({url: '/data'}).then(...);
   // The HTTP methods to be sent with the request.
   headers: { 'some': 'header' },
 
-  // The data to base64 encode and send in the body of the request.
+  // The data send in the body of the request. Objects will be converted to JSON.
```

Comment:
perhaps let's just say:

```
The data to send in the body of the request.
```

Reply:
Hmm. I was trying to capture the underlying behaviour in a way that would make sense to the end developer. It seems at the moment there is some special handling for streams and that Javascript objects are converted to JSON. 

Although perhaps you are right that simpler is better. I have so far found that whilst this library is pretty good, the documentation tends to not always match the behaviour so I'd rather it was correct and vague than incorrect and precise.

Reply:
@strideynet I see your point, perhaps just fix the typo `The data send` to `The data to send`, and perhaps:

```
Data objects will be serialized as JSON
```

:point_up: streams are technically an object as well, so something that calls out the fact that this refers to the behavior of [POJOs](https://en.wikipedia.org/wiki/Plain_old_Java_object) (I think perhaps more folks will understand "data object" than POJO).

Reply:
That's fair enough. I'll incorporate that change.